app/src/dbRequest.js

```diff
@@ -0,0 +1,57 @@
+//Task Requests
+export const getTasks = async (user) => {
+  const response = await fetch("/api/tasks", {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      "timedown-user": user,
+    },
+  });
+  return response.json();
+};
+
+export const addTask = async (
+  userId,
+  dueDate,
+  estTime,
+  summary,
+  description,
+) => {
+  const response = await fetch("/api/tasks", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ userId, dueDate, estTime, summary, description }),
+  });
+
+  return getTasks(userId);
+};
+
+export const deleteTask = async (id) => {
+  const response = await fetch("/api/tasks/delete", {
+    method: "POST",
```

Comment:
Do you know if ExpressJS recognizes request types besides `GET` and `POST`?  Technically, a request which is intended to perform a delete action should use the `DELETE` request type.